### PR TITLE
New metrics: XOnBattery, XOffBattery, LastSelfTest, NominalPower

### PR DESCRIPTION
**Pull request by rschaeuble**

### CODE REVIEW

File: upscollector.go [hunk 1]
```diff
@@ -5,6 +5,7 @@ import (
 
 	"github.com/mdlayher/apcupsd"
 	"github.com/prometheus/client_golang/prometheus"
+	"time"
```
Comment (mdlayher) [inline upscollector.go]: Put this import up with `"log"` in the first block.

File: upscollector.go [hunk 2]
```diff
@@ -110,6 +115,34 @@ func NewUPSCollector(ss StatusSource) *UPSCollector {
 			nil,
 		),
 
+		LastTransferOnBattery: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, "", "apcupsd_last_transfer_on_battery"),
+			"Time of last transfer to battery since apcupsd startup",
```
Comment (mdlayher) [inline upscollector.go]: Add a period to all of these help text strings.

**Comment by rschaeuble** [comment]
Fixed both issues.

**Comment by mdlayher** [comment]
LGTM.

**Pull request merged**